### PR TITLE
Masking helper functions

### PR DESCRIPTION
**Pull request by xaviervia**

### CODE REVIEW

File: packages/msked/package.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "msked",
+  "version": "0.0.0",
+  "description": "Functions to operate with masked string values",
+  "keywords": ["masking", "input"],
+  "main": "src/index.ts",
```
Comment (deepsweet) [inline packages/msked/package.json]: is it ok that it will have only Node build?

Comment (xaviervia) [reply packages/msked/package.json]: Nope. You are right. I copied the wrong package. Thanks!

File: packages/msked/test/index.ts [hunk 2]
```diff
@@ -0,0 +1,300 @@
+import test from 'blue-tape'
+import { applyMask, getPositionInMasked, getPositionInValue, isLongerThanMask } from '..'
+
+test('getPositionInMasked', (t) => {
```
Comment (deepsweet) [inline packages/msked/test/index.ts]: so far we have a single stream of Tape test results for entire metarepo so it's a good idea to prefix tests like `msked: getPositionInMasked`

Comment (xaviervia) [reply packages/msked/test/index.ts]: Will do

**Comment by deepsweet** [comment]
also I'd squash it into a single 🐣 because technically there can't be a 🌱 feature for something that is not published yet.

**Comment by xaviervia** [comment]
I'll squash now when merging

**Pull request merged**